crates/bcr-wallet-core/scripts/test_mac.sh

```diff
@@ -0,0 +1,5 @@
+# Clang/LLVM issues on MAC
```

Comment:
Protom added some build and check commands for our wasm build to our [justfile](https://github.com/BitcreditProtocol/Bitcredit-Core/blob/master/justfile recently - maybe they can be useful here as well